src/components/TheSearchBar.vue

```diff
@@ -131,7 +131,7 @@ function isMicroblockId(query) {
     border: none;
     background-color: var(--color-white);
     margin-right: var(--space-1);
-    font-size: 14px;
+    font-size: 16px;
```

Comment:
Very small detail: I would leave 14px for desktop and set 16px only for mobile devices.

Reply:
good idea, adjusted